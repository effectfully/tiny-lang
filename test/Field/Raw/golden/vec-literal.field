{T, F, T, F, T, F}