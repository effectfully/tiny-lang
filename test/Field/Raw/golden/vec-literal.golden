EConst (Some (UniVal Vector [True,False,True,False,True,False]))